azure/folder.go

```diff
@@ -254,3 +245,21 @@ func getUploadStreamToBlockBlobOptions(settings map[string]string) azblob.Upload
 	}
 	return azblob.UploadStreamToBlockBlobOptions{MaxBuffers: maxBuffers, BufferSize: bufferSize}
 }
+
+// Function will get environment's name and return string with the environment's Azure storage account endpoint suffix.
+// Expected names AzureUSGovernmentCloud, AzureChinaCloud, AzureGermanCloud. If any other name is used the func will return
+// the Azure storage account endpoint suffix for AzurePublicCloud.
+func getStorageEndpointSuffix(environmentName string) string {
+	var storageEndpointSuffix string
+	switch environmentName {
+	case "AzureUSGovernmentCloud":
```

Comment:
Replace "AzureUSGovernmentCloud" with azure. azure.USGovernmentCloud.Name?

Reply:
For each case name, I used the environment name that Microsoft is returning when use [Azure Instance Metadata Service](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/instance-metadata-service?tabs=linux). Also they are the same names as the ones used in the variables of the [module](https://pkg.go.dev/github.com/Azure/go-autorest/autorest/azure#pkg-variables) that I am using.

The plan is to have the user that is configuring wal-g, to set/select the environment name that want to use. 
I believe the names I have are simpler to use vs `azure.USGovernmentCloud.Name`

Reply:
It looks like they are the same:
https://github.com/Azure/go-autorest/blob/c0fe8b0fea3d7344cc0a3e332c1bf28006c7fd83/autorest/azure/environments.go#L84

Reply:
Took me a few mins to realize but in the end I understand what you mean. I made the [change](https://github.com/wal-g/storages/pull/43/commits/ea752ec5f3f8335cd58a7d179da4d841c3337730 )